### PR TITLE
Add Job detail page for the companies

### PR DESCRIPTION
**Pull request by miralsuthar**
This PR close issue #18 and provide the job detail page for the companies who list their jobs on the job page.

### CODE REVIEW

File: client/pages/jobs/[company-name]/[jobTitle].tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import JobDetail from '../../../components/jobs/JobDetail';
+
+export default function JobDetails() {
```
Comment (angeljgomezc) [inline client/pages/jobs/[company-name]/[jobTitle].tsx:3]: What is the purpose of `JobDetails` here? What are we doing with this component?

File: client/components/jobs/JobDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        borderTopWidth={2}
+        align="center"
+      >
+        <ApplyButton w="70%">Apply For This Job</ApplyButton>
+      </Stack>
+    </Stack>
+  );
+}
+
+const Detail = ({
```
Comment (angeljgomezc) [inline client/components/jobs/JobDetail.tsx:175]: What is this for?

**Review by angeljgomezc — CHANGES_REQUESTED** [review]

File: client/components/jobs/JobDetail.tsx (hunk 2 above)
Comment (miralsuthar) [reply client/components/jobs/JobDetail.tsx:175]: the Detail component is built by with-heart to give detail about the job in the ```job card``` by simply using the detail component you can write the title and its below text simply.
image

File: client/pages/jobs/[company-name]/[jobTitle].tsx (hunk 1 above)
Comment (miralsuthar) [reply client/pages/jobs/[company-name]/[jobTitle].tsx:3]: It's just for the single export function for the page to have.

**Comment by Dhaiwat10** [comment]
image

Vercel logs, build failing

**Comment by with-heart** [comment]
> image
> 
> Vercel logs, build failing

Could someone explain what the issue was and how we resolved it please?

**Comment by angeljgomezc** [comment]
> > image
> > 
> > Vercel logs, build failing
> 
> Could someone explain what the issue was and how we resolved it please? 

This was a weird issue. I think relates to workspaces and yarn. At first, we couldn't deploy with the old Vercel config as it was throwing the error in the screenshot.

Then we decided to change the config in Vercel. From having the "Root Directory" on `client` and no hardcoded build commands , to using the default root directory of the repo and instead changing "Output Directory" to `client/.next`. We also added `yarn build` as the build command

The previous Vercel config had `client` as root directory and no other configuration or command

**Pull request merged**